docs/_data/docs_nav.yml

```diff
@@ -0,0 +1,36 @@
+- title: Getting Started
+  docs:
+    - link: /docs/
+    - link: /docs/ruby-101/
```

Comment:
Why not have a simple array of links instead..?
```yaml
docs:
  - /docs/
  - /docs/ruby-101/
  - # and so on..
```

Reply:
I think this approach is congruent to [`docs/_data/primary_nav.yml`](https://github.com/jekyll/jekyll/pull/7205/files/657995b1304629998f4c73e89ce2794bddf2bafc#diff-862fd229cd6af6922b29b6c1b8caa9ce), allowing for things like `title` or `show_on_mobile` to be set for each link.